src/lua/LuaPropertyMap.h

```diff
@@ -0,0 +1,11 @@
+// Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
```

Comment:
// Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt